physical/physical.go

```diff
@@ -88,6 +88,7 @@ var BuiltinBackends = map[string]Factory{
 	"dynamodb":  newDynamoDBBackend,
 	"etcd":      newEtcdBackend,
 	"mysql":     newMySQLBackend,
+	"postgres":  newPostgreSQLBackend,
```

Comment:
I guess go with PostgreSQL as that's the official name of the project. I only use `postgres` because it's easier to say.